### ISSUE TITLE
Add database connection helper command

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-local-server#1]
It's easy to connect to a chassis db for example so it'd be useful to be able to do the same here. The command should be named `composer server db`.

Acceptance criteria:

- [x] `composer server db` should log the user into mysql on the db server
- [x] `composer server db spf` should generate and open an SPF file for sequel pro (also add an alias of `sequel`)
- [x] `composer server db info` should print out db connection details
- [x] Update command help output
- [x] Document db command in a new file `docs/database.md`

`composer server db info` output template:

```
Root password: wordpress

Database: wordpress
Username: wordpress
Password: wordpress

Host: 0.0.0.0
Port: 31232

Version: MySQL 5.7
```

Bonus

- [ ] `composer server db workbench` Windows support for MySQL Workbench

**Comment by roborourke** [comment]
This one is a bit meatier but will be helpful. A few more notes to assist in development:

- See the `composer server logs` command for a reference on how to check for subcommands and arguments
- Example `.spf` template: https://github.com/Chassis/SequelPro/blob/master/template.spf
- Get the mapped db ports using
  ```
  docker ps --format "{{.Ports}}" --filter name=altis-dev_db_1
  # -> 33060/tcp, 0.0.0.0:32123->3306/tcp
  ```
  `0.0.0.0` is the host and `32123` in this case is the port on the host you can access it at
- username and pass is `wordpress`, `wordpress`

**Comment by roborourke** [comment]
Closed by #122
